google/cloud/bigquery/v2/minimal/internal/job_response.cc

```diff
@@ -51,6 +54,17 @@ StatusOr<nlohmann::json> parse_json(std::string const& payload) {
   return json;
 }
 
+std::string DebugJobsString(std::vector<ListFormatJob> const& jobs,
```

Comment:
It looks like this, and all the helper functions above, should be in an anonymous namespace.